### DISCUSSION TITLE
Add example of using q(quat) in the snapshot interpolation example

### DISCUSSION THREAD

**Discussion by ip3ly5** [geckosio/geckos.io#145] [Ideas]
Snapshot interpolation has been working brilliantly for me when using single number values, 
but when I try to use quaternion I get the error "No lerp method "linear" found!".

I think it's an issue with my code but showing how to use quat/deg/rad in the example would go a long way to help others who might also be struggling.

**Comment by yandeu** [comment]
There is [an example](https://github.com/geckosio/snapshot-interpolation#degrees-radians-and-quaternions). Doesn't it work?

**Reply by ip3ly5** [reply]
Sorry I should've been more clear, I've been using the examples in your link with little success, I could not get them to work. I was thinking that showing its usage in [this example project](https://github.com/geckosio/snapshot-interpolation/tree/master/example) would be very useful.

**Reply by yandeu** [reply]
I have visualized it and it looks like there is a issue with the calculation.
See https://enable3d.io/examples/standalone/snapshot-interpolation.html

Edit:
I made a typo in the example. Now it works
